README.md

```diff
@@ -1,70 +1,157 @@
 
-### Analyzing the Bundle Size
+<!-- GETTING STARTED -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Making a Progressive Web App
+Git clone https://github.com/SAZZAD1Q2/GlobeTrekker.git
+Run in your browser
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Prerequisites
 
-### Advanced Configuration
+In order to run this project you need:
+Git
+An IDE like vs code . bracket etc.
+Code Editor
+Browser
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Setup
 
-### Deployment
+Clone this repository to your desired folder: https://github.com/SAZZAD1Q2/GlobeTrekker.git
+Opent it to your code editor
+open it then to your browser.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
-### `npm run build` fails to minify
+### Install
+
+Install this project with:
+Github or git or vs code. For this open your terminal and proceed some steps:
+a. Install node package using the command : npm install
+b. Then run the website using the command : npm start
+c. Test the code using jest snapShots and for this use the Command : npx jest
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### deployment
+
+You may use the online version of my work. Here is the link of online version: [Link](https://globe-trekker.vercel.app/)
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/SAZZAD1Q2/GlobeTrekker.git)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sazzad3y/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+[Link of the recording](https://www.loom.com/share/6d9aa4fb7042449fafeae3d76544c9ad?sid=972552a3-cb8a-41ef-b837-2a5ed6e51796)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions are always welcome. Feel free to make your issue.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+If you like this project. Give me star. please spread it out to your network.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse and the creator who generated the idea of the app and design. 
```

Comment:
- [ ] Please add the link to the website then it looks professional for now you just give credit to the creator but you didn't tell who is the creator of this design so kindly add the link :slightly_smiling_face: 

**Example**
![Screenshot from 2023-08-03 15-54-04](https://github.com/SAZZAD1Q2/GlobeTrekker/assets/45226848/752691ed-1893-4ea9-acff-ce36b571f2f2)

Reply:
Thank you again for your nice advice. How can I get the link of the creator?